Fully Functioning GUI Bingo

RoseMatt319 opened the pull request:
Created a hints menu that displays bingo, but not anything else yet due to need to reuse code currently not in this branch.
The hints menu has its own taskbar space and a recognizable icon.

RoseMatt319 committed:
Fully Functioning GUI Bingo

Only bingo for now, the hint for first two letters will be implemented next.

Hints now has its own taskbar tab which properly displays the bingo.

kingchewie reviewed (dismissed):
def letters is good: tested on both our machines/branches
I dont fully understand gui So ill do my best here:
Looks like a hints display window was created (fine as best i can tell)
linked to action in the same manner as all others (fine)
function to call to create hints (or maybe to pop up the hints window) (looks good as best i can tell)
IDK what getHintsUIDisplay does
hintDialog looks like its responsible for linking windo to hints text
Not totally sure the purpose of getBing(puzzle); the code is fine but it is not used anywhere

RoseMatt319 committed:
Merge branch 'develop' into gui-hint-window

RoseMatt319 committed:
Finished GUI Hints View

GUI hints now shows everything necessary.
Also, added perfect pangrams as shown in the example.

kingchewie reviewed (approved):
countPerfect looks good, Same as countPan but with stipulation that the pan is exactly 7 in length
added perfect pangrams output ot CLI
gui window adjustmnets
